modules/microshift-rhoai-con.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * microshift_ai/microshift-rhoai.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-rhoai-con_{context}"]
+= How {rhoai-full} works in {microshift-short}
+
+Edge deployments are where data happens and decisions need to be made. You can use {rhoai-full} ({rhoai}) to integrate a fleet of {microshift-short}-driven edge devices into the artificial intelligence and machine learning (AI/ML) operations cycle.
+
+{rhoai} is a platform for data scientists and developers of AI/ML applications. Develop, train, and test an AI model in the cloud or data center using {rhoai}, then run your model in your edge deployments on {microshift-short}.
+
+After you deploy your AI model, application data can be sent to the model. In turn, the model learns to make highly accurate predictions. Over time, your AI model can make data-driven decisions without a human user. This is an ideal scenario for edge applications where interaction with an administrator is naturally limited.
```

Comment:
Can model serving process learn and improve its prediction capability while running? Asking because this approach is new to me. My understanding is that learning (training) and making predictions (inferencing) are two complete separate steps.

Reply:
No, you're right.